appveyor.yml

```diff
@@ -1,19 +1,19 @@
-branches:
-  only:
-    - master
+branches:	
```

Comment:
The only changes to this file are extra white spaces on each line?

Reply:
Hah it is, but it wasn't intentional.  Just from me copy/pasting from the GitHub UI.  I'll completely revert it now.